index.html

```diff
@@ -235,6 +235,15 @@ <h3 class="form-text">I'm always interested in hearing about new projects, so if
         </ul>
       </form>
     </section>
+    <footer>
+      <ul class="socia_icons">
+        <li><a href="#"><img src="images/github.png" alt=""></a></li>
+        <li><a href="#"><img src="images/Linkedin-icon.png" alt=""></a></li>
+        <li><a href="#"><img src="images/peace-icon.png" alt=""></a></li>
+        <li><a href="#"><img src="images/tweeter-icon.png" alt=""></a></li>
+        <li><a href="#"><img src="images/Vector(2).png" alt=""></a></li>
```

Comment:
Kindly add alternative text in the alt control rather than leave it empty as this will enable the viewer to see what the slot is about should the image not display.